### PR TITLE
Badges!!!

### PR DESCRIPTION
**Pull request by pdw207**
## Summary

You can run code coverage reports on `yarn test` and travis should run on master and take care of badge updates or view on the [dashboard](https://codeclimate.com/github/ResistanceCalendar/resistance-calendar-frontend).

[3.31 GPA  with 65% code coverage](https://codeclimate.com/github/ResistanceCalendar/resistance-calendar-frontend) 
## README
screen shot 2017-04-28 at 3 34 20 pm

### CODE REVIEW

**Comment by InciteDemocracy** [comment]
@pdw207 you are an admin, there must be some other reason you're blocked up.

**Comment by pdw207** [comment]
@InciteDemocracy I think I need to be an admin of the organization not just the repo.
screen shot 2017-04-27 at 9 15 17 am

**Comment by InciteDemocracy** [comment]
Aha, ok added you as an owner.

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "dev": "webpack-dev-server",
     "start": " serve -s dist",
     "heroku-postbuild": "rm -rf dist && NODE_ENV=production webpack -p --progress --colors",
-    "test": "jest --config=testConfig/jest.config.json",
+    "test": "jest --config=testConfig/jest.config.json --coverage",
```
Comment (pfarnach) [inline package.json]: Should we worry about `yarn run test:watch` generating a report on each file save?

**Pull request merged**